docs/scripts/fledge_plugin_list

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+function table {
+	list="$1"
+	title=$2
+	tableType=$3
+	echo ".. list-table:: $title" >> $output
+	echo "    :widths: 20 50" >> $output
+	echo "    :header-rows: 1" >> $output
+	echo "    :stub-columns: 1" >> $output
+	echo "" >> $output
+	echo "    * - Name" >> $output
+	echo "      - Description" >> $output
+	for repo in $list
+	do
+		product=`echo $repo | sed -e 's/-.*//'`
+		if [ $product = "foglamp" ]; then
+			org=dianomic
+		else
+			org=fledge-iot
+		fi
+		type=`echo $repo | sed -e 's/fledge-//' -e 's/foglamp-//' -e 's/-.*//'`
+		name=`echo $repo | sed -e 's/fledge-//' -e 's/foglamp-//' -e "s/${type}-//"`
+		echo $repo, $name, $type >>/tmp/repos
+		if [ "$type" = "$tableType" ]; then
+			description=`curl -s -H "$header" https://api.github.com/repos/$org/$repo | jq '.description' | sed -e 's/"//g'`
+			echo "    * - $name" >> $output
+			echo "      - $description" >> $output
+		fi
+	done
+	echo "" >> $output
+}
+
+if [ ! -f ~/.gittoken ]; then
+	echo "You must have a git access token in the file ~/.gittoken"
+	exit 1
+fi
+
+output=$1
+
+token=`cat ~/.gittoken`
+header="Authorization: token $token" 
+fledgeRepos=`curl -s -H "$header" https://api.github.com/orgs/fledge-iot/repos\?per_page=1000 | jq '.[].name' | sed -e 's/"//g' | sort`
```

Comment:
1) We can refactor the `jq` as we want only plugins to be fetched out.

```
jq '.[] | select(.name | contains("fledge-south") or contains("fledge-north-") or contains("fledge-filter-") or contains("fledge-rule-") or contains("fledge-notify-")) | .name' | tr -d '""'
```
2) For description info to be fetched out, explicit curl request is NOT needed if add `.description` into above jq but yes it needs some modification in `$REPOSITORIES list & function table` as well as below has description as well.

https://github.com/fledge-iot/fledge/blob/65b8e428f48bdd10990fa3b5adeffd0dfa92f849/docs/scripts/fledge_plugin_list#L25

Reply:
We could refactor in lots of ways, but frankly we run this infrequently and it is not worth the effort.